python/foglamp/plugins/south/cc2650/cc2650.py

```diff
@@ -335,12 +334,9 @@ def plugin_poll(handle):
                 'key': str(uuid.uuid4()),
                 'readings': {"percentage": battery_level}
             })
-    except RuntimeError as ex:
+    except (Exception, RuntimeError, pexpect.exceptions.TIMEOUT) as ex:
         _plugin_restart(bluetooth_adr)
-        raise exceptions.QuietError(str(ex))
-    except (Exception, pexpect.exceptions.TIMEOUT) as ex:
-        _plugin_restart(bluetooth_adr)
-        raise exceptions.DataRetrievalError(str(ex))
```

Comment:
Also remove the reference from docstring

Reply:
done